src/sidebar/host-config.js

```diff
@@ -50,13 +50,45 @@ export default function hostPageConfig(window) {
     'usernameUrl',
   ];
 
+  // Coerces specific config values that we expect to be of specific types
+  // but could be provided in another.
+  const translations = {
+    // Ensures requestConfigFromFrame.ancestorLevel is integer or NaN
+    requestConfigFromFrame: value => {
+      if (typeof value === 'string') {
+        return value;
+      } else {
+        return {
+          ...value,
+          ancestorLevel: Math.round(Number(value.ancestorLevel)),
+        };
+      }
+    },
+    // Ensures openSidebar is boolean
+    openSidebar: value => {
+      if (value === 'false') {
```

Comment:
Could we do the equivalent of `value.lower()` here to catch title case 'False'? This is a value which could easily be emitted by Python.

Reply:
Will do